source/PaNET_metadata.ttl

```diff
@@ -65,5 +65,5 @@ This project was undertaken under ExPaNDS WP3.2 (https://expands.eu/)
 The ontology is can be found here: https://github.com/ExPaNDS-eu/ExPaNDS-experimental-techniques-ontology
 
 Photon and neutron PaN ontologies developed under ExPaNDS are documented here: https://doi.org/10.5281/zenodo.4806026"^^xsd:string ;
-owl:versionInfo "1.0"^^xsd:string ;
+owl:versionInfo "1.0.1"^^xsd:string ;
```

Comment:
Suggest decoupling changes targeting contents from the release process.

One way to do this (while avoiding any ambiguity in a release) is to use a version with a placeholder indicating the current version isn't an actual release; for example, `1.0.1-SNAPSHOT`.  The `-SNAPSHOT` suffix indicates that the repo (in its current state) is not `1.0.1` proper.

The release process then creates two commits: one that puts the repo into the state for the release, the other prepares the repo for the next development cycle.  The first commit would change the version from `1.0.1-SNAPSHOT` to `1.0.1` (potentially among other changes) while the second commit would change the version from `1.0.1` to `1.0.2-SNAPSHOT`.  The first commit would then be tagged with the appropriate tag (`1.0.1`).

With this patch accepted as-is, anyone could check out the repository and believe they are working with PaNET v1.0.1, which might not be the case.